_topic_maps/_topic_map_rosa.yml

```diff
@@ -162,6 +162,8 @@ Topics:
     File: rosa-installing-rosa
   - Name: Creating a ROSA cluster without AWS STS
     File: rosa-creating-cluster
+  - Name: Configuring a private cluster
```

Comment:
Did we update the module includes listed with the new assembly name? `rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-private-cluster.adoc`